src/Select/index.tsx

```diff
@@ -265,6 +265,7 @@ export const Select: React.FC<Props> = ({
   defaultValue,
   disabled = false,
   feel,
+  color,
```

Comment:
I feel like it'd be more intuitive to set the default here
```suggestion
  color = colors.white,
```